### DISCUSSION TITLE
Missing Copy Scripts

### DISCUSSION THREAD

**Discussion by stonefox1958** [5axes/Calibration-Shapes#54] [Q&A] [answered]
Use Cura 4.10.0

In "Extensions - Part for Calibration" the menu item "Copy Scripts" is missing 
![Cura plugin](https://user-images.githubusercontent.com/8201781/127685716-893cc66f-d6fc-4ac8-9832-b854a35a4431.jpg)

**Comment by 5axes** [comment, marked as answer]
From release 4.9 don't need to use this function anymore. So this function is disable in the last release
